test/behavior.zig

```diff
@@ -105,6 +105,8 @@ test {
             if (builtin.zig_backend != .stage2_c) {
                 // Tests that pass for stage1 and the llvm backend.
                 _ = @import("behavior/atomics.zig");
+                _ = @import("behavior/bitreverse.zig");
```

Comment:
Would you be fine moving these test files all the way up and adding `return error.SkipZigTest` for each test case individually for each backend which doesn't implement this yet? I realise it's a little bit of typing but frankly, it would make life easier as I have to do that for any new bit of functionality added to `x86_64` backend for instance.

Reply:
No problem at all - good to know what the latest convention is, too

Reply:
Thanks!